### ISSUE TITLE
[Bug][a11y]: Card action menu is not accessible using the keyboard in small variant 

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3389]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Card

### Expected behavior

Visually hidden action menu button should display when it receives focus.

```css
:host([size="s"]) .actions,
:host([size="s"]) .quick-actions {
    visibility: visible;
}
```

### Actual behavior

The action menu button within a small Card, has `visibility: hidden` except on `:hover` or `:focus-within`, which means the action menu button cannot receive focus unless the Card is hovered.

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error

**Comment by najikahalsema** [comment]
We should clarify the expected behaviour for this scenario. Looking at [this example](https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/card--small-quiet), it does look like a strange interaction.
